src/ets_lru.erl

```diff
@@ -173,10 +179,11 @@ handle_call({insert, Key, Val}, _From, St) ->
             true = ets:delete(St#st.atimes, ATime),
             true = ets:insert(St#st.atimes, {NewATime, Key});
         [] ->
-            Entry = #entry{key=Key, val=Val, atime=NewATime, ctime=NewATime},
+            NewCTime = element(1, NewATime),
+            Entry = #entry{key=Key, val=Val, atime=NewATime, ctime=NewCTime},
             true = ets:insert(St#st.objects, Entry),
             true = ets:insert(St#st.atimes, {NewATime, Key}),
-            true = ets:insert(St#st.ctimes, {NewATime, Key})
+            true = ets:insert(St#st.ctimes, {NewCTime, Key})
```

Comment:
NewCTime is Erlang's monotonic time. According to documentation:
> consecutive calls to erlang:monotonic_time/0 can produce the same result.

Reply:
I must have thought for some reason that `ctime` table entries were not required to be unique, and that we could save some memory by not storing that extra integer. However, after looking at [this code](https://github.com/apache/couchdb-ets-lru/blob/master/src/ets_lru.erl#L286) and [also this](https://github.com/apache/couchdb-ets-lru/blob/master/src/ets_lru.erl#L306) it does indeed appear that `ctime` entries are expected to be unique, and so like `atime`, also of type `strict_monotonic_time()`. It's been changed in the latest commit.